src/main/java/hw3/repository/UserRepositoryJdbcImpl.java

```diff
@@ -0,0 +1,23 @@
+package hw3.repository;
+import hw3.entity.User;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+
+public class UserRepositoryJdbcImpl implements UserRepository {
```

Comment:
You are not using JDBC in this implementation
```suggestion
public class UserRepositoryImpl implements UserRepository {
```